CRM/Admin/Form/ScheduleReminders.php

```diff
@@ -350,6 +350,13 @@ public static function formRule($fields, $files, $self) {
       $errors[$recipientKind[$fields['recipient']]['target_id']] = ts('If "Also include" or "Limit to" are selected, you must specify at least one %1', array(1 => $recipientKind[$fields['recipient']]['name']));
     }
 
+    //CRM-21523
+    if (!empty($fields['is_repeat']) &&
+      (empty($fields['repetition_frequency_interval']) || ($fields['end_frequency_interval'] == NULL))
```

Comment:
@lcdservices because you used `==` instead of `===` I don't think those two conditions are actually different.
Php thinks that `0 == NULL` but not `0 === NULL`.

Reply:
I agree that it seems like we should use the strict operator, but it doesn't work when you do. I believe that's because the values we receive are cast as strings, not integers. If we use the strict operator an empty value for the end_frequency_interval field does not trigger the validation.